src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
```

Comment:
I don't see how StrictComparer works for the test. This stub is an equivalent to

$this->getMock('ValueComparer')->method('whatever')->willReturn($someBool);

As this is a stub, it would be weird to use it in production code. It's meant for use in tests